### DISCUSSION TITLE
Published site updates are here!

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#536] [Announcements]
Hi everyone! You may have seen previews of this over the last few weeks — and today we’re excited to let you know that your published GitBook content has gotten some major upgrades.

![Group 1](https://github.com/GitbookIO/community/assets/42930383/115843e1-c7ac-43be-8729-051124108c8d)

Here’s what you’ll see:

🏃  **Improved performance** – Pages should load faster, feel snappier, and generally give readers a better experience.
A new look and feel – Your docs now have a more modern design and a streamlined layout that makes everything easier to browse and read.

🔍  **Search across spaces** – Published a collection full of spaces? Now users can find information across them all from one search bar.

📄  **Better PDF exports** – The PDF export experience is now faster and easier than ever — and it produces even better results.

🔒  **Simpler visitor authentication** – You can now set up authenticated access for published docs using our Auth0, Azure AD and Okta integrations.

**What’s next?**
In the next few weeks we’ll have some more big news to share about published documentation. For now, make sure you check out all the updates in our latest blog post, and head over to GitBook to take a look at them for yourself!

**Comment by egorkarimov** [comment]
Great that GitBook team has energy to improve its platform, however... I was dissapointed by the new look and feel. There are many visiual issues: 
- fonts are not that sharp anymore;
- padding, margins, text lines height changed and there is less space to rest for my eyes;
- embedded content is broken (will add a screenshot);
- borders I do not need: around images, embedded content, etc;
- code blocks  have rows highligthing with hovered line borders; =/
- gray color of the elements, and background leaves the dirty design impression;
- inline images in links are opened when clicked!
- overall `techie` impression on published docs, while I am doing a humantarian project: math or geometry-like cells when things are loading;

I appreciate Gitbook a lot. I would love to see relevant improvements. I am ready to pay for it to have collaboration with my team. 
Please do something about this. Thank you!

<img width="1168" alt="Screenshot 2024-03-21 at 22 09 20" src="https://github.com/GitbookIO/community/assets/3321584/c599b16f-549a-49dd-8fb0-1fc61b4dab8e">

**Reply by addisonschultz** [reply]
Thanks for the feedback @egorkarimov - We appreciate you using GitBook with us and for your projects 😄 Thanks for the feedback on the styles - they're valid points we'll take into consideration as we continue to iterate on the published content aspect.

Can you send our support team more info on the embedded issue? Sounds like something we can fix quickly!

You can reach us here: support (@) gitbook.com

**Reply by egorkarimov** [reply]
Thanks for your reply! Appreciate! 

I have just sent info about the embedded content issue to the support team via Gitbook chat.

I understand that my feedback is not about major bugs or functional issues. It is more about style and perception.
Though, I consider it significant because of a visual impression our content leaves in visitors.
Typography is crucial aspect of UX, IMHO. Just compare:

### Used to be:

<img width="930" alt="Screenshot 2024-03-22 at 14 37 56" src="https://github.com/GitbookIO/community/assets/3321584/a5e1c677-b218-4dd6-b9b9-bafc9ae070a5">

### With current update:

<img width="930" alt="Screenshot 2024-03-22 at 14 47 05" src="https://github.com/GitbookIO/community/assets/3321584/e5786881-3137-44c7-8e59-796417458e68">

In this new version H2 titles thickness is different without a reason, or possibility to control it. Font size of the Hint block is different from the paragraph text. Line height is not that good anymore. Oh, I want my clean and neat docs back! 😭

Though, fair to say that some changes are better. Content of Table is better, Expandable Blocks are clickable now.

Thanks.

**Comment by nikitin-pro** [comment]
Hello, I actually like restyling, except checkboxes. In editor they are looking fine, but on published pages one can barely distinguish between checked and unchecked items (try it on mobile in sunny day while walking).
![image](https://github.com/GitbookIO/community/assets/95876501/a53dae60-9bf7-4a17-bf24-25d2f492d06e)

**Reply by egorkarimov** [reply]
Also cards text is barely readable now. I need to bend my neck to read these sma-a-a-all letters.
<img width="509" alt="Screenshot 2024-03-23 at 22 59 58" src="https://github.com/GitbookIO/community/assets/3321584/c3929644-243c-4f54-a374-96cc210577b6">
